homeassistant/components/shelly/binary_sensor.py

```diff
@@ -272,6 +272,26 @@ def __init__(
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_class=RpcBluTrvBinarySensor,
     ),
+    "flood": RpcBinarySensorDescription(
+        key="flood",
+        sub_key="alarm",
+        name="Flood",
+        device_class=BinarySensorDeviceClass.MOISTURE,
+    ),
+    "mute": RpcBinarySensorDescription(
+        key="flood",
+        sub_key="mute",
+        name="Mute",
+        device_class=BinarySensorDeviceClass.SOUND,
```

Comment:
As I mentioned earlier, I think that the `BinarySensorDeviceClass.SOUND` states (`Detected` and `Clear`) do not fit the entity named `Mute`. My proposition: `Alarm sound` and states `On`/`Off`.

Reply:
Done